util/from-random/src/lib.rs

```diff
@@ -21,3 +24,17 @@ impl<const N: usize> FromRandom for [u8; N] {
         result
     }
 }
+
+pub fn random_bytes_vec<R: CryptoRng + RngCore>(num_bytes: usize, csprng: &mut R) -> Vec<u8> {
```

Comment:
I don't think these belong here. The purpose of this crate is to just provide the trait.

Reply:
My thinking is that these are only used in tests at the moment, and as such belong in `mc-util-test-helper` but this is a nit not worth spending time on.

Reply:
My thinking is that these are only used in tests at the moment, and as such belong in `mc-util-test-helper` but this is a nit not worth spending time on.